packages/core/test/slider/sliderTests.tsx

```diff
@@ -43,6 +43,14 @@ describe("<Slider>", () => {
         assert.lengthOf(wrapper.find(`.${Classes.SLIDER}-label`), 7);
     });
 
+    it("labelStepSize=0 renders only min and max axis labels", () => {
+        const wrapper = renderSlider(<Slider min={10} max={20} labelStepSize={0}/>);
+        const labels = wrapper.find(`.${Classes.SLIDER}-label`);
+        assert.lengthOf(labels, 3);
```

Comment:
look for the label inside the `pt-slider-axis`  instead? it's odd that the description is for the min, max but the test is also checking for the label on the handle